src/Nethermind/Nethermind.Crypto/Keccak512.cs

```diff
@@ -62,9 +63,9 @@ public static Keccak512 Compute(byte[]? input)
             return InternalCompute(input);
         }
 
-        public static uint[] ComputeToUInts(byte[] input)
+        public static uint[] ComputeToUInts(ReadOnlySpan<byte> input)
         {
-            if (input is null || input.Length == 0)
+            if (input.IsNull() || input.Length == 0)
```

Comment:
`.IsNull` will also be fine with `.Length == 0` for spans
```suggestion
            if (input.Length == 0)
```